src/redux/categories/categories.js

```diff
@@ -0,0 +1,15 @@
+// import { createReducer, createAction } from '@reduxjs/toolkit';
```

Comment:
- [ ]  @EddyShimwa kindly remove the commented code, it's not good practice to have commented code in your codebase.